### PR TITLE
Add circuit sapling react app

### PR DESCRIPTION
**Pull request by eloaverona**
This PR:
- Sets ups a docker network for demos and test.
- Adds a basic react app for the circuit sapling

#### To test:
1.  Run:
```
docker-compose -f docker/docker-compose.yaml up --build
```
2. Go to `http://localhost:3030/circuits`:
Screen Shot 2020-04-08 at 9 06 37 AM

### CODE REVIEW

File: docker/docker-compose.yaml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+# limitations under the License.
+# ------------------------------------------------------------------------------
+
+version: '3.6'
+
+volumes:
+  contracts:
+  key-registry:
+
+services:
```
Comment (dplumb94) [inline docker/docker-compose.yaml:22]: This looks good but we might want to add another node so we can test both parties proposing/voting on a circuit.

File: sapling-dev-server/Dockerfile [hunk 2]
```diff
@@ -21,6 +21,10 @@ RUN npm config set unsafe-perm true
 
 COPY . .
 
+RUN cd /saplings/circuits && \
+  npm install && \
+  yarn deploy
```
Comment (dplumb94) [inline sapling-dev-server/Dockerfile]: I believe it is this way in the Grid saplings, but to be consistent you probably want to do `npm run deploy`. We had to switch to npm due to git dependencies not working properly with yarn.

File: sapling-dev-server/userSaplings [hunk 3]
```diff
@@ -1 +1,14 @@
...
+    "displayName": "Circuits",
+    "namespace": "circuits",
+    "runtimeFiles": [
+      "localhost:3030/sapling-dev-server/circuits/js/circuits.js"
+    ],
+    "styleFiles": [
+      "localhost:3030/sapling-dev-server/circuits/css/circuits.css"
+    ],
+    "workerFiles": [],
+    "icon": "https://via.placeholder.com/48/333333/FFFFFF?text=C"
```
Comment (dplumb94) [inline sapling-dev-server/userSaplings]: I think we should skip the placeholder. My browser is blocking it now too. Davey might have an icon we can use instead. I have a PR up in grid that shows how to include an icon in the sapling dev server.

**Review by dplumb94 — CHANGES_REQUESTED** [review]

File: docker/docker-compose.yaml (hunk 1 above)
Comment (eloaverona) [reply docker/docker-compose.yaml:22]: I added a second node, ui and db.

File: sapling-dev-server/Dockerfile (hunk 2 above)
Comment (eloaverona) [reply sapling-dev-server/Dockerfile]: Updated

File: sapling-dev-server/userSaplings (hunk 3 above)
Comment (eloaverona) [reply sapling-dev-server/userSaplings]: Addressed

**Review by dplumb94 — APPROVED** [review]

**Review by agunde406 — APPROVED** [review]
Tested locally

**Pull request merged**